src/quoteservice/app/routes.php

```diff
@@ -0,0 +1,27 @@
+<?php
+declare(strict_types=1);
+
+use OpenTelemetry\API\Trace\AbstractSpan;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\App;
+
+return function (App $app) {
+    $app->get('/getquote', function (Request $request, Response $response) {
+        $span = AbstractSpan::getCurrent();
```

Comment:
this just does everything in the one root span. that's ok and will work, but the output might look better if it created a child span here?